Logging is configured.
======================================================= test session starts ========================================================
platform win32 -- Python 3.12.4, pytest-8.3.2, pluggy-1.5.0
rootdir: C:\Users\150194\OneDrive - Arrow Electronics, Inc\Documents\Python\pytest-project
configfile: pytest.ini
plugins: html-4.1.1, html-reporter-0.2.9, metadata-3.1.1
collected 9 items

tests/api/test_api_todo_pixegami.py::test_api_get getting response message

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:40 INFO getting response message
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "GET / HTTP/11" 200 39
PASSED                                                                                                                        [ 11%]
tests/api/test_api_todo_pixegami.py::test_api_put_create_task Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "PUT /create-task HTTP/11" 200 187
Created new task with id: task_f197b23acdb94e14b4269e439970148d

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:42 INFO Created new task with id: task_f197b23acdb94e14b4269e439970148d
PASSED                                                                                                                        [ 22%]
tests/api/test_api_todo_pixegami.py::test_api_put_create_task_invalid_data_type Creating task with invalid data type.

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:42 INFO Creating task with invalid data type.
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "PUT /create-task HTTP/11" 422 111
PASSED                                                                                                                        [ 33%]
tests/api/test_api_todo_pixegami.py::test_api_put_create_task_missing_required_field Creating task with missing one of required field.

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:42 INFO Creating task with missing one of required field.
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "PUT /create-task HTTP/11" 422 91
PASSED                                                                                                                        [ 44%]
tests/api/test_api_todo_pixegami.py::test_api_put_create_task_invalid_creationtime SKIPPED (unconditional skip)               [ 55%]
tests/api/test_api_todo_pixegami.py::test_api_get_task_by_id Get task by task id: task_f197b23acdb94e14b4269e439970148d

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:43 INFO Get task by task id: task_f197b23acdb94e14b4269e439970148d
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "GET /get-task/task_f197b23acdb94e14b4269e439970148d HTTP/11" 200 178
PASSED                                                                                                                        [ 66%]
tests/api/test_api_todo_pixegami.py::test_api_get_task_by_user_id Get task by user id: Test User 1

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:44 INFO Get task by user id: Test User 1
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "GET /list-tasks/Test%20User%201 HTTP/11" 200 1040
PASSED                                                                                                                        [ 77%]
tests/api/test_api_todo_pixegami.py::test_api_update_task Update task with id: task_f197b23acdb94e14b4269e439970148d

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:45 INFO Update task with id: task_f197b23acdb94e14b4269e439970148d
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "PUT /update-task HTTP/11" 200 31
Get task with id: task_id 5 and validate update task
2024-08-07 23:34:47 INFO Get task with id: task_id 5 and validate update task
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "GET /get-task/task_id%205 HTTP/11" 200 76
PASSED                                                                                                                        [ 88%]
tests/api/test_api_todo_pixegami.py::test_api_del_task_by_id Delete task with id: task_f197b23acdb94e14b4269e439970148d

---------------------------------------------------------- live log call -----------------------------------------------------------
2024-08-07 23:34:48 INFO Delete task with id: task_f197b23acdb94e14b4269e439970148d
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "DELETE /delete-task/task_f197b23acdb94e14b4269e439970148d HTTP/11" 200 59
Validate task is deleted or not.
2024-08-07 23:34:49 INFO Validate task is deleted or not.
Starting new HTTPS connection (1): todo.pixegami.io:443
https://todo.pixegami.io:443 "GET /get-task/task_f197b23acdb94e14b4269e439970148d HTTP/11" 404 65
PASSED                                                                                                                        [100%]

================================================== 8 passed, 1 skipped in 11.73s ===================================================
